Python/context.c

```diff
@@ -723,7 +727,9 @@ PyTypeObject PyContext_Type = {
 static int
 contextvar_set(PyContextVar *var, PyObject *val)
 {
+#ifndef Py_GIL_DISABLED
     var->var_cached = NULL;
+#endif
     PyThreadState *ts = _PyThreadState_GET();
```

Comment:
Consider moving this down to the `#ifndef` below to avoid the "unused variable warning"